### PR TITLE
CAL-455 Replace pom properties previously removed

### PR DESCRIPTION
**Pull request by mcalcote**
#### What does this PR do?
Replace properties that had been moved to the parent pom.

#### Who is reviewing it? 
@tbatie @ryeats @derekwilhelm @coyotesqrl @oconnormi @lessarderic @paouelle @Kjames5269 

#### How should this be tested?
Build Alliance and ensure no dependency version are undefined. It my be easiest to confirm if ddf-parent is build locally with all the properties removed and then update the root pom to look at that snapshot. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/alliance/pom.xml [hunk 1]
```diff
@@ -28,7 +28,6 @@
     <packaging>pom</packaging>
     <properties>
         <setup.folder>target/dependencies</setup.folder>
-        <opendj-embedded.app.version>1.3.3</opendj-embedded.app.version>
```
Comment (paouelle) [inline distribution/alliance/pom.xml]: 👍

File: pom.xml [hunk 2]
```diff
@@ -91,6 +91,89 @@
...
+        <project.report.output.directory>project-info</project.report.output.directory>
+
+        <!--Documentation properties-->
+        <asciidoctor.maven.plugin.version>1.5.6</asciidoctor.maven.plugin.version>
+        <asciidoctor.source.highlighter>coderay</asciidoctor.source.highlighter>
+        <asciidoctorj.diagram.version>1.5.4.1</asciidoctorj.diagram.version>
+        <asciidoctorj.pdf.version>1.5.0-alpha.16</asciidoctorj.pdf.version>
+        <asciidoctorj.version>1.5.6</asciidoctorj.version>
+
+        <awaitility.version>3.1.2</awaitility.version>
```
Comment (paouelle) [inline pom.xml]: ✏️ This is a test library

File: pom.xml [hunk 3]
```diff
@@ -91,6 +91,89 @@
         <gem-maven-plugin.version>1.0.5</gem-maven-plugin.version>
```
Comment (paouelle) [inline pom.xml]: ❓ Should we add a plugin properties section and move the other down below to their corresponding sections?

Comment (mcalcote) [reply pom.xml]: Will add

File: pom.xml [hunk 4]
```diff
@@ -91,6 +91,89 @@
...
+        <netty.version>4.1.24.Final</netty.version>
+        <node.version>v8.9.4</node.version>
+        <ogc-filter-v_1_1_0-schema.version>1.1.0_5</ogc-filter-v_1_1_0-schema.version>
+        <opendj-embedded.app.version>1.3.3</opendj-embedded.app.version>
+        <org.slf4j.version>1.7.1</org.slf4j.version>
+        <pax.exam.version>4.12.0</pax.exam.version>
+        <pax.url.version>2.5.4</pax.url.version>
+        <plexus-compiler-javac-errorprone.version>2.8.2</plexus-compiler-javac-errorprone.version>
+        <require-css.version>0.1.10</require-css.version>
+        <surefire.version>2.20.1</surefire.version>
```
[the remark this answers is not in the document]
Comment (mcalcote) [reply pom.xml]: Sure

**Comment by mcalcote** [comment]
build now

**Comment by cxbot1** [comment]
Internal build has been scheduled, your results will be available at build completion.

**Comment by cxbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.codice.org/job/Alliance-Jobs/job/pr/job/Linux/1035/
:white_check_mark: JOB SUCCESS

**Pull request merged**